docs/user_guide.rst

```diff
@@ -222,6 +222,20 @@ a ``.saucelabs`` configuration file in the working directory or your home
 directory, or by setting the ``SAUCELABS_USERNAME`` and ``SAUCELABS_API_KEY``
 environment variables.
 
+Alternatively, when using `Jenkins CI`_ declarative pipelines,
+credentials can be set as environment variables as follows:
+
+.. code-block:: groovy
+
+  environment {
+    SAUCELABS = credentials('SAUCELABS')
+  }
+
+For more information see Jenkins `docs`_.
+
+.. _Jenkins CI: <https://jenkins.io/>
+.. _docs: <https://jenkins.io/doc/pipeline/tour/environment/>
```

Comment:
dammit, I tried them out in an online rst parser. But maybe the merge/rebase debacle screwed me.

Still not 100% confident with git :(

Sorry.